data/custom-warnings.json

```diff
@@ -1,58 +1,94 @@
-[
-  {
-    "vaultAddress": "0x0F359FD18BDa75e9c49bC027E7da59a4b01BF32a",
-    "chainId": 1,
-    "level": "red",
-    "metadata": {
-      "content": "The risk curator has executed a forced removal of the sdeUSD/USDC market. The vault's share price decreased proportionally, resulting in a loss for affected depositors. For more information, please refer to the Vault Curator's updates."
+{
```

Comment:
We expect an array in the api so this will break prd and staging. We could update the API but how useful is it to have separated warnings for you?

Also we need to expect `uniqueKey` for markets (currently expect marketId but this is tech debt)